vaxrank/report.py

```diff
@@ -63,85 +67,110 @@ def compute_template_data(
         if len(vaccine_peptides) == 0:
             logger.info("Skipping %s, no vaccine peptides", variant_short_description)
             continue
-        gene_name = vaccine_peptides[0].mutant_protein_fragment.gene_name
-        peptides = []
-        variant_dict = {
-            'num': i + 1,
-            'short_description': variant_short_description,
-            'gene_name': gene_name,
-            'top_score': vaccine_peptides[0].combined_score,
-            'predicted_effect': variants_to_top_coding_effect_dict.get(variant),
-            'reads_supporting_variant_allele':
-                vaccine_peptides[0].mutant_protein_fragment.n_alt_reads,
-            'reads_supporting_reference_allele':
-                vaccine_peptides[0].mutant_protein_fragment.n_ref_reads,
-            'reads_supporting_other_alleles':
-                vaccine_peptides[0].mutant_protein_fragment.n_other_reads,
-            'peptides': peptides,
-        }
 
-        # compile peptide info
-        for j, vaccine_peptide in enumerate(vaccine_peptides):
-            epitopes = []
-            # compile epitope info
-            for epitope_prediction in vaccine_peptide.epitope_predictions:
-                if epitope_prediction.overlaps_mutation:
-                    score = epitope_prediction.logistic_score()
-                    epitope_dict = {
-                        'sequence': epitope_prediction.peptide_sequence,
-                        'ic50': epitope_prediction.ic50,
-                        'normalized_binding_score': score,
-                        'allele': epitope_prediction.allele,
-                    }
-                    epitopes.append(epitope_dict)
+        ############################### Variant info ###############################
```

Comment:
What about factoring out all these Foo Infos into functions like `foo_info(...)`. Would be a bit easier to read and follow that way, I think?